src/components/forms/ComboBox/ComboBox.test.tsx

```diff
@@ -0,0 +1,1111 @@
+        key: 'ArrowDown',
+      })
+
+      expect(getByTestId('combo-box-option-yuzu')).toHaveFocus()
+    })
+
+    xit('pressing tab once in the input with a selected option focuses the clear button', () => {
+      // TODO: 🐛  Currently first tab opens list to selected option and second tab selects first option
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          defaultValue="banana"
+          onChange={jest.fn()}
+        />
+      )
+
+      // focus banana
+      userEvent.click(getByTestId('combo-box-input'))
+      userEvent.tab()
+
+      // tab once
+      userEvent.tab()
+
+      expect(getByTestId('combo-box-clear-button')).toHaveFocus()
+    })
+
+    xit('pressing tab twice in the input with a selected option focuses the selected option', () => {
+      /* TODO: 🐛  Currently first tab opens list to selected option and second tab selects first option 
+        Expected behavior: 
+        1. we should be moving the focus to the selected option when its first open
+        2. 1st tab goes to clear, second tab goes to selected option 
+      */
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          defaultValue="banana"
+          onChange={jest.fn()}
+        />
+      )
+
+      // focus banana
+      userEvent.click(getByTestId('combo-box-input'))
+      userEvent.tab()
+
+      // tab twice
+      userEvent.tab()
+      userEvent.tab()
+
+      expect(getByTestId('combo-box-option-banana')).toHaveFocus()
+    })
+
+    it('focuses the next option when an option is selected and down arrow is pressed', () => {
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          defaultValue="mango"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      userEvent.click(getByTestId('combo-box-input'))
+      fireEvent.keyDown(getByTestId('combo-box-option-mango'), {
+        key: 'ArrowDown',
+      })
+
+      expect(getByTestId('combo-box-option-mangosteen')).toHaveFocus()
+    })
+
+    it('does not close menu when an option is selected and the first option is focused and up arrow is pressed', () => {
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          defaultValue="apricot"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      // Apple is the item at top of list
+      fireEvent.focus(getByTestId('combo-box-option-apple'))
+      fireEvent.keyDown(getByTestId('combo-box-option-apple'), {
+        key: 'ArrowUp',
+      })
+
+      expect(getByTestId('combo-box-option-apple')).toHaveFocus()
+      expect(getByTestId('combo-box-option-list')).toBeVisible()
```

Comment:
this is not the behavior I'm seeing.. in Chrome when I have selected Apricot and the Apple option is focused and I press the up arrow, the menu closes

Reply:
🐛 This is a bug and this test should be failing.  Changing `fireEvent.focus` to `userEvent.hover(getByTestId('combo-box-option-apple'))` makes it fail as it should.  Great catch!